app-crypt/tpm2-tss/tpm2-tss-3.2.0.ebuild

```diff
@@ -11,30 +11,32 @@ SRC_URI="https://github.com/tpm2-software/${PN}/releases/download/${PV}/${P}.tar
 
 LICENSE="BSD-2"
 SLOT="0"
-KEYWORDS="amd64 arm arm64 ppc64 x86"
-IUSE="doc +fapi gcrypt +openssl static-libs test"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+IUSE="doc +fapi  +openssl mbedtls static-libs test"
 
 RESTRICT="!test? ( test )"
 
-REQUIRED_USE="^^ ( gcrypt openssl )
-		fapi? ( openssl !gcrypt )"
+REQUIRED_USE="^^ ( mbedtls openssl )
+		fapi? ( openssl !mbedtls )"
 
 RDEPEND="acct-group/tss
 	acct-user/tss
-	fapi? (
-		dev-libs/json-c
-		net-misc/curl
-	)
-	gcrypt? ( dev-libs/libgcrypt:0= )
-	openssl? ( dev-libs/openssl:0= )"
+	fapi? ( dev-libs/json-c
```

Comment:
Not subscribing to json-c subslot? Do we not link against it?

Reply:
I did not know it had a subslot. Fixed.